src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
```

Comment:
You don't need flexbox here, as long as you have nothing to align inside the `stars__star` div :)